src/pages/Profile.tsx

```diff
@@ -110,41 +147,37 @@ const Profile = () => {
             </IconButton>
             <Box>
               <Typography sx={styles.displayName}>
-                {user.displayName}
+                {profileContents.displayName}
               </Typography>
               <Typography sx={styles.tweetCount}>
                 {profileContents.postCount} Tweets
               </Typography>
             </Box>
           </Box>
           <Box>
-            <Box
-              component="img"
-              sx={styles.banner}
-              src={"/blue.jpg"}
-              alt="Temp"
-            />
+            <Box sx={styles.banner} />
             <Box sx={styles.profileContent}>
               <Box sx={styles.avatarContainer}>
-                <Avatar
-                  alt="Profile Picture"
-                  src={"/rock.jpg"}
-                  sx={styles.avatar}
-                />
-                <Button
-                  variant="contained"
-                  startIcon={<EditIcon />}
-                  sx={styles.editProfileButton}
-                >
-                  Edit Profile
-                </Button>
+                <Avatar sx={styles.avatar} />
+                {profileContents.username === user.username ? (
+                  <Button
+                    size="small"
+                    variant="outlined"
+                    startIcon={<EditIcon />}
+                    sx={styles.editProfileButton}
+                  >
+                    Edit Profile
+                  </Button>
+                ) : (
+                  <FollowButton />
+                )}
```

Comment:
Another kinda unrelated thing but the edit profile button shows for a brief moment when you first load the page. I think we already have a ticket for figuring out how to handle and `undefined` user, but we need loaders on a lot of things anyways so that can be another ticket

Reply:
Yeah definitely need loaders, I'll save this for that ticket

Reply:
Great, I made a draft ticket and threw it in the backlog